ultravox/training/configs/release_config.yaml

```diff
@@ -0,0 +1,45 @@
+# SLM with ultravox & llama3.1, trained wtih knowledge distillation.
```

Comment:
Will we update this file only when we do an official release?

Reply:
Yes, this is the plan, if we want to make the training recipe available to the community.